app/uk/gov/hmrc/fileupload/read/envelope/model.scala

```diff
@@ -49,13 +51,18 @@ case class File(fileId: FileId,
 
 object Envelope {
 
+  val defaultMaxCapacity = 100
```

Comment:
Yep. Think we had talked about putting them in some constants file previously, but config makes more sense.